tests/internal/log.c

```diff
@@ -0,0 +1,124 @@
+#include <fluent-bit/flb_sds.h>
+#include <fluent-bit/flb_mem.h>
+#include <fluent-bit/flb_log.h>
+
+#include "flb_tests_internal.h"
+
+#define TIMEOUT             5
+#define TEST_RECORD_01      "this is a test message"
```

Comment:
nitpick: Maybe, we might have to include emoji characters to test non-alphabetic characters?

```diff
-#define TEST_RECORD_01      "this is a test message"
+#define TEST_RECORD_01      "this is a test message😃"
```